spec/components/schemas/content/voice/call.ts

```diff
@@ -0,0 +1,60 @@
+import { SchemaObject } from 'openapi3-ts';
+import { ref as baseRef } from './../base';
+import { ref as ttsRef } from '../tts';
+import { ref as audioFileRef } from '../audiofile';
+import { createComponentRef } from '../../../../../utils/ref';
+
+const text: SchemaObject = {
+  type: 'object',
+  allOf: [{
+    $ref: baseRef,
+  }, {
+    type: 'object',
+    properties: {
+      type: {
+        type: 'string',
+        example: 'call',
+      },
+      audio: {
+        description: 'An array of audio actions',
+        type: 'array',
+        items: {
+          anyOf: [{
+            $ref: ttsRef,
+          }, {
+            $ref: audioFileRef,
+          }],
+          discriminator: {
+            propertyName: 'type',
+            mapping: {
+              tts: ttsRef,
+              file: audioFileRef,
+            },
+          },
+        },
+      },
+      transferTo: {
+        description: 'Transfer or call to another number',
+        type: 'string',
+        example: '5511999999999',
+      },
+      recordAudio: {
+        description: 'To record the call',
```

Comment:
Talvez explicar melhor (e assumindo que lembro certo como funciona),
"Records the call when enabled. The recording audio file will be made available on a status webhook call once the call is completed"

(e só é possível se tivermos implementado o status também, o que acho que não ainda né? talvez pra uma próxima essa)